### PR TITLE
Split building v2 and v3 VSIXes into separate commands

### PR DESCRIPTION
**Pull request by mavasani**
Latest SWIX plugin doesn't support building them with the same msbuild command.

### CODE REVIEW

**Comment by mavasani** [comment]
@srivatsn can you review

File: build/build.proj [hunk 1]
```diff
@@ -88,7 +80,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildModernVsixPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildModernVsixPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;Test" />
```
Comment (srivatsn) [inline build/build.proj]: But this means that Jenkins won't build the modern vsixes right? can we instead make this conditionally exclude? i.e Microsoft can call build.cmd /excludemodernvsix and build.cmd /modernvsixesonly but build.cmd builds all?

Comment (mavasani) [inline build/build.proj]: Jenkins will anyway not build modern VSIXes as that target is conditioned on running in microbuild.

Comment (srivatsn) [inline build/build.proj]: Ah ok. We need to move to the swix SDK package to build them in Jenkins but that's a different workitem.

**Pull request merged**